helm-chart/images/binderhub/requirements.txt

```diff
@@ -130,7 +131,7 @@ pyasn1-modules==0.3.0
     # via google-auth
 pycparser==2.21
     # via cffi
-pycurl==7.45.2
+pycurl==7.45.3
```

Comment:
This causes the tests to fail https://github.com/pycurl/pycurl/issues/834

Reply:
If I've understood the issue, adding `--no-binary=pycurl` ought to fix it

Reply:
I failed to get it working in z2jh @minrk, but we have build stages etc and its a bit complicated.

https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/3365

> I tried to do `--no-binary=pycurl` etc, but we have multistage builds etc making things a bit complicated, and I didn't got it working.
> 
> Practically, I tried adding the `--no-binary=pycurl` to the `pip wheel` command in our build stage.

Reply:
I saw that. I believe https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/3371 is the missing piece there.

The approach taken here (`pip install --no-deps *.whl`) already avoids the problem solved by `--no-index` there, because it doesn't try to recalculate _what_ to install from a requirements file, instead saying "install all these wheel files, don't look anywhere else or think too much about it."

Reply:
Looks like it didn't work! Will investigate further.